### PR TITLE
Feat: wishlist api integration

### PR DESCRIPTION
**Pull request by ArjunGTX**
- integrated wishlist api
- fixed cart duplicate items
- removed unwanted files

### CODE REVIEW

File: src/pages/cart/components/CartCard.jsx [hunk 1]
```diff
@@ -28,6 +33,30 @@ export const CartCard = ({ item }) => {
...
+    try {
+      const { status, data } = await deleteFromWishlist(
+        auth.encodedToken,
+        item._id
+      );
+      if (status !== 200) return;
+      setWishlist(data.wishlist);
+      return;
+    } catch (error) {
+      console.log(error);
```
Comment (sharathnair9999) [inline src/pages/cart/components/CartCard.jsx:56]: u r catching the error but just consoling it. you could show it up as an error message or just remove the console log

**Review by sharathnair9999 — APPROVED** [review]
Suggested trivial changes. You are good to go

File: src/pages/cart/components/CartCard.jsx (hunk 1 above)
Comment (ArjunGTX) [reply src/pages/cart/components/CartCard.jsx:56]: Yes, that will be done in the future through alert mesaages.

**Review by tanishk15sharma — APPROVED** [review]
your code is looking good , I have learnt good practice from ur code u are breaking axios requests in fucntions .

**Pull request merged**